homeassistant/components/vesync/light.py

```diff
@@ -54,6 +48,8 @@ def _setup_entities(devices, async_add_entities):
             entities.append(VeSyncDimmableLightHA(dev))
         elif DEV_TYPE_TO_HA.get(dev.device_type) in ("bulb-tunable-white",):
             entities.append(VeSyncTunableWhiteLightHA(dev))
+        elif hasattr(dev, "night_light") and dev.night_light:
+            entities.append(VeSyncNightLightHA(dev))
```

Comment:
Maybe the night light is only available on humidifiers, but you can add support for the night light separately from adding the humidifier platform.

Reply:
Same as the sensors, humidifiers aren't detected as devices right now. But, I can split this change out and update lights after the humidifier support exists